rllib/evaluation/rollout_worker.py

```diff
@@ -1816,15 +1843,18 @@ def _build_policy_map(
                     # the running of these preprocessors.
                     self.preprocessors[name] = preprocessor
 
-            # Create the actual policy object.
-            self.policy_map.create_policy(
-                name,
-                policy_spec.policy_class,
-                obs_space,
-                policy_spec.action_space,
-                policy_spec.config,  # overrides.
-                merged_conf,
-            )
+            if policy is not None:
```

Comment:
Depending on whether the policy is given as an already instantiated object or not, use either create_policy() or insert_policy(). Note that `create_policy` also uses `insert_policy` internally now.